parsers/ENTSOE.py

```diff
@@ -1,5 +1,23 @@
 #!/usr/bin/env python3
 
+"""
+Parser that uses the ENTSOE API to return the following data types.
+
+Consumption
+Production
+Exchanges
+Exchange Forecast
+Day-ahead Price
+Generation Forecast
+Consumption Forecast
+
+The zones below use methods from this parser.
+
+['AT', 'BA', 'BE', 'BG', 'CH', 'CZ', 'DE', 'DK', 'DK-DK1', 'DK-DK2', 'EE', 'ES',
```

Comment:
Can we leave this out? The info is already in zones.json

Reply:
The list or the whole module docstring?

Reply:
just the list (I prefer having only one place to maintain instead of two).
Good initiative by the way!

Reply:
Thanks, I'll change it.